src/index.html

```diff
@@ -13,5 +13,145 @@
   <body>
     <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="http://localhost:5500/api" method="post">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div class="form-field">     
+          Surname:   
+          <input 
+          type="text" 
+          name="username" 
+          required
+          pattern="\w+"
+          >
```

Comment:
```suggestion
          <input 
            type="text" 
            name="username" 
            required
            pattern="\w+"
          >
```
fix code syle

Reply:
checklist
<img width="901" alt="image" src="https://user-images.githubusercontent.com/102411130/220549684-0d60aad3-9b4b-4f45-b542-05159dae83f7.png">

Reply:
make all such cases like in GOOD EXAMPLE